include/openthread.h

```diff
@@ -1802,7 +1673,7 @@ OTAPI void OTCALL otPlatformReset(otInstance *aInstance);
  *
  * @param[in]  aInstance A pointer to an OpenThread instance.
  */
-void otFactoryReset(otInstance *aInstance);
+OTAPI void OTCALL otFactoryReset(otInstance *aInstance);
```

Comment:
Windows is exposing the exact same functions. The only thing different is the calling convention, since they are exposed from a dynamic link library; not static. The goal here is that code written to control OpenThread shouldn't have to change at all. It would just have to change what they link to. If we made a separate header it would just create additional work to keep them in sync and they would just be exact copies of each other.

All the actual OS specific code has already been separated into the examples folder. The Windows specific DLL implementation is under examples/drivers/windows/otapi.

I thought we had resolved all these issues when all this code was initially added.